PDFDocument error in trimet-mod-otp

When using otp-rr as a dependency in trimet-mod-otp (see https://github.com/ibi-group/trimet-mod-otp/tree/otp-rr-3.1.0), there are webpack build issues due to the pdfkit dependency added in #378:

```
ERROR in ./node_modules/png-js/png-node.js
Module not found: Error: Can't resolve 'fs' in '/Users/binhdam/git/trimet-mod-otp/node_modules/png-js'
 @ ./node_modules/png-js/png-node.js 21:11-24
 @ ./node_modules/pdfkit/js/pdfkit.es5.js
 @ ./node_modules/otp-react-redux/build/util/PDFDocumentWithTables.js
 @ ./node_modules/otp-react-redux/build/util/mailables.js
 @ ./node_modules/otp-react-redux/build/components/admin/mailables-window.js
 @ ./node_modules/otp-react-redux/build/index.js
 @ ./lib/main.js
```

--- a/lib/util/PDFDocumentWithTables.js
+++ b/lib/util/PDFDocumentWithTables.js
@@ -1,6 +1,6 @@
 'use strict'
 
-const PDFDocument = require('pdfkit')
+import PDFDocument from 'pdfkit'
 
 /**
  * This class extends the pdfkit functionality to provide a tables method, which

